src/components/TextInput.tsx

```diff
@@ -14,46 +13,36 @@ type TextInputProps = {
   value: string;
   type: string;
   isError: boolean;
-  backgroundColor?: string;
   className?: string;
   onChange: (value: string) => void;
 };
```

Comment:
instead of using all these props, try extending the default input props, and only add the ones that are custom such as label, isError etc.

Reply:
you'll need to make this an interface to do that